packages/react/src/container/container.tsx

```diff
@@ -47,61 +47,72 @@ type NativeAttrs = Omit<React.HTMLAttributes<unknown>, keyof Props>;
 
 export type ContainerProps = Props & typeof defaultProps & NativeAttrs;
 
-const Container: React.FC<React.PropsWithChildren<ContainerProps>> = ({
-  xs,
-  sm,
-  md,
-  lg,
-  xl,
-  wrap,
-  gap,
-  as,
-  display,
-  justify,
-  direction,
-  alignItems,
-  alignContent,
-  children,
-  responsive,
-  fluid,
-  css,
-  ...props
-}) => {
-  const gapUnit = useMemo(() => {
-    return `calc(${gap} * $space$sm)`;
-  }, [gap]);
+const Container = React.forwardRef<
+  HTMLElement,
```

Comment:
@jrgarciadev since you allow this component to be polymorphic (because of the `as` prop), I have added a more ambiguous type for html element. Do you agree?